lib/index.d.ts

```diff
@@ -20,6 +20,11 @@ declare namespace stringify {
     }
 
     type Input = any[] | PlainObject<any>
+    
+    interface ColumnOptions {
```

Comment:
I think this interface would be better named `ColumnOption` (in singular).
```suggestion
    interface ColumnOption {
```